DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsControllerTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.TrackingSystem.CentreConfiguration
+{
+    using System.Security.Claims;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using DigitalLearningSolutions.Web.Controllers.TrackingSystem;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using FluentAssertions.Execution;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using NUnit.Framework;
+
+    public class RegistrationPromptsControllerTests
+    {
+        private ICentresDataService centresDataService;
+        private ICustomPromptsService customPromptsService;
+        private CentreConfigurationController centreConfigurationController;
+        private ISession httpContextSession;
+        private const int CentreId = 2;
+
+        [SetUp]
+        public void Setup()
+        {
+            centresDataService = A.Fake<ICentresDataService>();
+            customPromptsService = A.Fake<ICustomPromptsService>();
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new[]
+            {
+                new Claim("UserCentreID", CentreId.ToString())
+            }, "mock"));
+            httpContextSession = new MockHttpContextSession();

```

Comment:
Are user and httpContextSession used anymore?

Also, the warnings around the private declarations can be silenced by writing:
`private ICentresDataService centresDataService = null!; `
etc to tell the compiler the value is intended to be null at that stage.